### ISSUE TITLE
Keter needs process <1.4

### ISSUE THREAD

**Issue by mitchellwrosen** [snoyberg/keter#147]
Keter is failing to deploy my app:

```
2016-06-06 12:40:29.12: Data.Conduit.Process.Unix.monitorProcess: Data/Conduit/Process/Unix.hs:(207,42)-(219,13): Missing field in record construction child_group
```

Due to:

https://github.com/snoyberg/keter/blob/master/Data/Conduit/Process/Unix.hs#L207

(missing `child_group` and `child_user` fields)

**Comment by tolysz** [comment]
What should we do? Make upper upper bound in the hackage for currently published package and fix it for the next version with the higher lower bound for `process`?

**Comment by mitchellwrosen** [comment]
Something like that. This is actually a super scary case that I hadn't previously thought of, and makes me seriously skeptical that missing upper bounds are a good idea, since the old code happily compiles (albeit with a warning).

**Comment by tolysz** [comment]
@snoyberg Would it be possible to edit the package (at)Hackage and set `process` upper-bound, and then work on an patch.
OR: set the bound  here, re-publish package, and then (if ever) start working on patch to support newer `process`?

**Comment by snoyberg** [comment]
If someone wants to edit the Hackage metadata, I can give them maintainer access (though this should really be @creichert's call). Adding support for newer process should be an easy matter of either:

1. Using CPP to add the field in question, or
2. Avoid using the `CreateProcess` constructor directly, and instead use the `proc` function

**Comment by creichert** [comment]
@tolysz I can update the metadata, what versions need `process<1.4`, is that the only change?

If someone wants to submit a PR, i'll gladly review and merge.

**Comment by tolysz** [comment]
There is a PR fixing all, but for the one already publish on Hackage, the `process<1.3` is the correct limit.

**Comment by creichert** [comment]
@tolysz Just made an edit, can you review: https://hackage.haskell.org/package/keter-1.4.3.1/keter.cabal

**Comment by creichert** [comment]
I just uploaded a new version: https://hackage.haskell.org/package/keter-1.4.3.2

This includes @tolysz fixes in #152. I also

If I missed something, please mention it and I will fix it and upload a new version.

Thanks!
